checkov/common/runners/base_runner.py

```diff
@@ -12,11 +12,43 @@
 
 class BaseRunner(ABC):
     check_type = ""
+    definitions = None
+    context = None
+    breadcrumbs = None
+    external_registries = None
+    graph_manager = None
+    graph_registry = None
 
     @abstractmethod
     def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=RunnerFilter(), collect_skip_comments=True):
         pass
 
+    def set_external_data(self, definitions: Optional[Dict], context: Optional[Dict], breadcrumbs: Optional[Dict]):
+        self.definitions = definitions
+        self.context = context
+        self.breadcrumbs = breadcrumbs
+
+    def load_external_checks(self, external_checks_dir: List[str]):
+        pass
+
+    def get_graph_checks_report(self, root_folder, runner_filter: RunnerFilter):
+        pass
+
+    def should_create_graph(self):
+        # This allows to check for a specific check type whether we want to build a graph
+        # The expected format of BUILD_GRAPH is BUILD_GRAPH=`cloudformation,terraform`
+        # A runner will limit building graph only if it uses this method
+        graphs_to_build = os.environ.get('BUILD_GRAPH')
+        return graphs_to_build and self.check_type in graphs_to_build.lower()
```

Comment:
@rotemavni Let's use a more explicit env var name please - `CHECKOV_CLOUDFORMATION_GRAPH` = true.

Reply:
@nimrodkor I thought that it could be a good practice to have this method reused for all frameworks in the future

Reply:
We want it to be temp, not consistent. Hopefully it will be removed soon, once we feel more comfortable with performance etc